README.md

```diff
@@ -41,39 +41,41 @@ This tool expects to run in an environment with the following attributes:
 
 Requesting a code review:
 
-    git appraise request
+    git review request
 
 Pushing code reviews to a remote:
 
-    git appraise push [<remote>]
+    git review push [<remote>]
 
 Pulling code reviews from a remote:
 
-    git appraise pull [<remote>]
+    git review pull [<remote>]
 
 Listing open code reviews:
 
-    git appraise list
+    git review
+    git review --list
```

Comment:
Given that `git branch --list` with no &lt;pattern&gt; supplied is essentially the exact same as `git branch`, and given that you'd like to keep inline with git itself, would you suggest that `git review --list` has an optional &lt;pattern&gt;, and if so what would the pattern be trying to match? Branches and tags can be referenced by something other than a hash. I don't believe a review can.